compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -2,7 +2,8 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 {-# LANGUAGE MultiWayIf #-}
-{-# LANGUAGE PatternSynonyms     #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE MonoLocalBinds #-}
```

Comment:
It seems a bit odd, to enable this just for this module. Can we just add the type signatures and leave it disabled or is there a specific reason for why we need this here?

Reply:
I can definitely remove it and keep the type signatures. :)